app/src/renderer/components/staking/PageCandidates.vue

```diff
@@ -44,8 +44,8 @@ export default {
   computed: {
     ...mapGetters(['candidates', 'filters', 'shoppingCart', 'user']),
     pageTitle () {
-      if (this.user.signedIn) return `Candidates (${this.candidatesNum} Selected)`
-      else return 'Candidates'
+      if (this.user.signedIn) return `Delegate (${this.candidatesNum} Selected)`
```

Comment:
i'm not sure about this. we need to confirm — but i think candidate is still the correct term.

Reply:
This is right, delegate is being used as a verb here. If that's not clear then maybe it could say `Delegate (X Candidates Selected)`.

Reply:
Should it say "Candidates Selected" or "Validators Selected"? I feel like having the "Candidates" label doesn't make sense. It will just cause users to be confused. If a user stakes their atoms to a Candidate instead of a Validator, what happens to their staking income? Only the percentage of atoms they stake to Validators will earn them staking atoms? If that is the case, who would ever stake atoms to a Candidate?

Reply:
Separately: the page name is Delegate, so the page title should also be Delegate. If the user clicks on "Delegate" and they go to the "Candidates" page, it's confusing. That's why I made this change. This relates to issue #128, where I made the case that we should settle on either "Stake" or "Delegate" as the page name instead of using both terms.

Reply:
Definitely "Candidates". You would delegate to a non-validator candidate if you want to help push them up into the top 100. There is probably a bias to delegating to validators, but users should be encouraged to delegate to the candidates they think would make best validators to ensure the network is secure.

Related to that, we will need to ensure the default sorting order doesn't make any candidates unfairly get more visibility (but it's not obvious what the best order is). Will probably also want to warn when delegating to a validator with too much voting power so they don't get an unsafe amount of power.

Reply:
Sounds good. Let's use "Delegate (X Candidates Selected)".

Yeah we'll have to experiment with different sorting methods.

Reply:
yep. good conversation here! the challenge is that validators and validator candidates are dynamic. so sometimes you're going to delegate to a validator while other times you could be delegating to a candidate. this could even change in the time it takes to type in the amount and submit...

i'm not sure the right way to think about this. and the language around it is challenging. an analogy...

- a politician is someone running for mayor (a candidate)
- a politician is someone who is the mayor (a validator)

they are both politicians. we don't have a word that covers both which i think is the source of the trouble.